### PR TITLE
Computer tweaks

### PR DESCRIPTION
**Pull request by Arrow768**
Pulling in some computer changes from bay

### CODE REVIEW

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -194,16 +194,20 @@
 #define NTNET_SYSTEMCONTROL 4		// Control of various systems, RCon, air alarm control, etc.
 
 // NTNet transfer speeds, used when downloading/uploading a file/program.
-#define NTNETSPEED_LOWSIGNAL 0.025	// GQ/s transfer speed when the device is wirelessly connected and on Low signal
-#define NTNETSPEED_HIGHSIGNAL 0.1	// GQ/s transfer speed when the device is wirelessly connected and on High signal
-#define NTNETSPEED_ETHERNET 0.5		// GQ/s transfer speed when the device is using wired connection
+#define NTNETSPEED_LOWSIGNAL 0.1	// GQ/s transfer speed when the device is wirelessly connected and on Low signal
+#define NTNETSPEED_HIGHSIGNAL 0.5	// GQ/s transfer speed when the device is wirelessly connected and on High signal
+#define NTNETSPEED_ETHERNET 1		// GQ/s transfer speed when the device is using wired connection
```
Comment (Arrow768) [inline code/__defines/misc.dm]: The speeds are subject to change during playtesting

File: code/game/machinery/recharger.dm [hunk 2]
```diff
@@ -111,59 +112,27 @@ obj/machinery/recharger/process()
...
+		else if(istype(charging, /obj/item/weapon/computer_hardware/battery_module))
+			var/obj/item/weapon/computer_hardware/battery_module/BM = charging
+			cell = BM.battery
+		if(istype(cell, /obj/item/weapon/cell))
+			var/obj/item/weapon/cell/C = cell
 			if(!C.fully_charged())
 				icon_state = icon_state_charging
-				C.give(active_power_usage*CELLRATE*charging_efficiency)
+				C.give(active_power_usage*CELLRATE)
 				update_use_power(2)
```
Comment (Arrow768) [inline code/game/machinery/recharger.dm]: This needs to be looked through and tested

File: code/modules/mob/living/silicon/ai/ai.dm [hunk 3]
```diff
@@ -276,13 +280,13 @@ var/list/ai_verbs_default = list(
...
-	power_channel = EQUIP
-	var/mob/living/silicon/ai/powered_ai = null
-	invisibility = 100
+// /obj/machinery/ai_powersupply
+// 	name="Power Supply"
+// 	active_power_usage=50000 // Station AIs use significant amounts of power. This, when combined with charged SMES should mean AI lasts for 1hr without external power.
+// 	use_power = 2
+// 	power_channel = EQUIP
+// 	var/mob/living/silicon/ai/powered_ai = null
+// 	invisibility = 100
```
Comment (Arrow768) [inline code/modules/mob/living/silicon/ai/ai.dm]: Delete that

File: code/modules/modular_computers/file_system/news_article.dm [hunk 4]
```diff
@@ -0,0 +1,43 @@
+// /data/ files store data in string format.
+// They don't contain other logic for now.
+/datum/computer_file/data/news_article
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/news_article.dm]: We should see if we can port that to be db based

File: code/modules/modular_computers/file_system/program.dm [hunk 5]
```diff
@@ -133,3 +148,10 @@
 		return 1
 	if(computer)
 		return computer.Topic(href, href_list)
+
+// // Relays the call to nano module, if we have one
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/program.dm]: Delete that

File: code/modules/modular_computers/file_system/programs/ai_restorer.dm [hunk 6]
```diff
@@ -0,0 +1,117 @@
+/datum/computer_file/program/aidiag
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/ai_restorer.dm]: Needs playtesting

File: code/modules/modular_computers/file_system/programs/antagonist/hacked_camera.dm [hunk 7]
```diff
@@ -0,0 +1,37 @@
+/datum/computer_file/program/camera_monitor/hacked
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/antagonist/hacked_camera.dm]: Playtesting

File: code/modules/modular_computers/file_system/programs/camera.dm [hunk 8]
```diff
@@ -0,0 +1,171 @@
+// Returns which access is relevant to passed network. Used by the program.
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/camera.dm]: Playtesting

File: code/modules/modular_computers/file_system/programs/comm.dm [hunk 9]
```diff
@@ -0,0 +1,311 @@
+#define STATE_DEFAULT	1
```
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/comm.dm]: Test it

File: code/_onclick/hud/screen_objects.dm [hunk 10]
```diff
@@ -427,20 +427,20 @@
...
 				AI.toggle_camera_light()
 
-		if("Crew Monitoring")
-			if(isAI(usr))
-				var/mob/living/silicon/ai/AI = usr
-				AI.subsystem_crew_monitor()
+		// if("Crew Monitoring")
+		// 	if(isAI(usr))
+		// 		var/mob/living/silicon/ai/AI = usr
+		// 		AI.subsystem_crew_monitor()
```
Comment (Arrow768) [inline code/_onclick/hud/screen_objects.dm]: This needs to be fixed

File: code/_onclick/hud/screen_objects.dm [hunk 11]
```diff
@@ -427,20 +427,20 @@
...
 				AI.show_station_manifest()
 
-		if("Show Alerts")
-			if(isAI(usr))
-				var/mob/living/silicon/ai/AI = usr
-				AI.subsystem_alarm_monitor()
+		// if("Show Alerts")
+		// 	if(isAI(usr))
+		// 		var/mob/living/silicon/ai/AI = usr
+		// 		AI.subsystem_alarm_monitor()
```
Comment (Arrow768) [inline code/_onclick/hud/screen_objects.dm]: This needs to be fixed

File: code/_onclick/hud/screen_objects.dm [hunk 12]
```diff
@@ -452,10 +452,10 @@
...
 				AI.ai_call_shuttle()
 
-		if("State Laws")
-			if(isAI(usr))
-				var/mob/living/silicon/ai/AI = usr
-				AI.subsystem_law_manager()
+		// if("State Laws")
+		// 	if(isAI(usr))
+		// 		var/mob/living/silicon/ai/AI = usr
+		// 		AI.subsystem_law_manager()
```
Comment (Arrow768) [inline code/_onclick/hud/screen_objects.dm]: This needs to be fixed

File: code/game/machinery/recharger.dm (hunk 2 above)
Comment (skull132) [inline code/game/machinery/recharger.dm]: I think charging efficiency was our thing?

File: code/__defines/misc.dm (hunk 1 above)
Comment (NanakoAC) [inline code/__defines/misc.dm]: I have to protest this, please leave it at 0.5 at most

File: code/game/machinery/recharger.dm (hunk 2 above)
Comment (NanakoAC) [inline code/game/machinery/recharger.dm]: it was indeed my thing. ensure its preserved

File: code/modules/modular_computers/file_system/programs/ai_restorer.dm (hunk 6 above)
Comment (NanakoAC) [inline code/modules/modular_computers/file_system/programs/ai_restorer.dm]: afaik this computer never served a purpose when it was standalone, i'm not sure what the point in making a program of it was

File: code/__defines/misc.dm (hunk 1 above)
Comment (NanakoAC) [inline code/__defines/misc.dm]: set them back to what they were before, those were already too fast.
We had a big argument about it yesterday and skull settled on the ethernet speed being 0.5, so preserve that ruling at least

File: code/modules/modular_computers/file_system/programs/ai_restorer.dm (hunk 6 above)
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/ai_restorer.dm]: Playtestet. Works.
Its used to restore a dead ai

File: code/__defines/misc.dm (hunk 1 above)
Comment (Arrow768) [inline code/__defines/misc.dm]: These values are the one from bay.
We´ll playtest them and then see how it goes.
Keep in mind that we have files up to 70GQ

File: code/modules/modular_computers/file_system/programs/comm.dm (hunk 9 above)
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/comm.dm]: Playtested. Works.

File: code/modules/modular_computers/file_system/programs/camera.dm (hunk 8 above)
Comment (Arrow768) [inline code/modules/modular_computers/file_system/programs/camera.dm]: Tested. Works.

File: code/__defines/machinery.dm [hunk 13]
```diff
@@ -46,6 +50,13 @@ var/global/defer_powernet_rebuild = 0      // True if net rebuild will be called
 #define NETWORK_SECURITY "Security"
 #define NETWORK_TELECOM "Tcomsat"
 #define NETWORK_THUNDER "Thunderdome"
+#define NETWORK_ALARM_ATMOS "Atmosphere Alarms"
+#define NETWORK_ALARM_POWER "Power Alarms"
+#define NETWORK_ALARM_FIRE "Fire Alarms"
+#define NETWORK_SUPPLY "Supply"
+#define NETWORK_EXPEDITION "Expedition"
+#define NETWORK_CALYPSO "Calypso"
```
Comment (NanakoAC) [inline code/__defines/machinery.dm]: what is calypso,and expedition

**Pull request merged**